src/app/components/Embeds/EmbedHtml/index.stories.tsx

```diff
@@ -0,0 +1,13 @@
+// Same as above but we will need React
```

Comment:
What does this comment mean?

Reply:
whoops leftover boilerplate comment, will remove https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/coding-standards-storybook--page#creating-story-for-a-component-with-variants